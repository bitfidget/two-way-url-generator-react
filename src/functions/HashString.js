import { Base64 } from 'js-base64'

const HashString = {
    decode: function (string) {
        return Base64.decode(string)
    },
    encode: function (string) {
        return Base64.encode(string)
    }
}

export default HashString