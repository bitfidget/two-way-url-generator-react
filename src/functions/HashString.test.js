import React from 'react'
import HashString from './HashString'

it('takes a string and returns a hash', () => {
    expect(HashString('hello', false)).toBe('aGVsbG8=')
})

it('takes a hash and returns a string', () => {
    expect(HashString('aGVsbG8=', true)).toBe('hello')
})
