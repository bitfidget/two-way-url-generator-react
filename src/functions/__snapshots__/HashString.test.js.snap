// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`takes a string or hash 1`] = `"orange"`;
